dcsazure_adls_to_adls_prof_pl/README.md

```diff
@@ -0,0 +1,203 @@
+those values needs to be escaped per the first point, so an escape character of `\` in your file would need to be `\\\\`
+3. It is assumed that `first_row_as_header` is true - but it is parameterized now in case additional support is added
+at a later time.
+4. It is important that you get the row delimiter correct. Files that don't have the correct row delimiter will yield
+strange column names and may fail the profiling dataflow (and therefore the pipeline).
+5. It is important that you get the column delimiter correct. Files that have the wrong column delimiter will yield
+strange column names and may fail the profiling dataflow (and therefore the pipeline).
+6. When getting the file metadata, the data set is not parameterized as there is a known issue with the `Get Metadata`
+step not respecting parameterized row delimiters. To work around this issue, the data set that is used to get the file
+metadata relies on the `Default` row delimiter, which matches `\r`, or `\n`, or `\r\n`.
+7. If it appears as though a directory's files have not been scanned, this often happens when the parameters are of
+`P_SUFFIX_DELIMITER_MAP` are incorrect, it is recommended to double-check this value.
+
+When scanning through folders, all subdirectories are expected to contain files wherein those files with the same
+suffix contain the same metadata as one another. Specifically, the header row of `prefix1_file1.csv` must be the same as
+`prefix2_file2.csv` if they are in the same subdirectory. (Note that this does not apply across file types, i.e. the
+header rows of `prefix1_file1.csv` and `prefix2_file2.txt` are considered to be different.) Support for scanning
+subdirectories with multiple file names with separate suffixes is managed by leveraging the parameter
+`P_SUB_DIRECTORY_WITH_MIXED_FILE_SCHEMAS`, which contains the names of subdirectories that need additional handling.
+The disambiguation of the schema is done by leveraging file prefixes (which may not be a subset of one another) and
+specifying how to interpret the prefixes using the parameter `P_MIXED_FILE_SCHEMA_DISAMBIGUATION`.
+
+The default value of `P_MIXED_FILE_SCHEMA_DISAMBIGUATION` can be more easily understood with formatting applied:
+```json
+{
+  "DCS_SAMPLE_PREFIX_": {
+    "suffixes": [
+      "csv",
+      "txt",
+      "NO_EXT"
+    ]
+  }
+}
+```
+
+This default value will likely not conflict with any of your profiling, but is provided as such to illustrate how to
+specify prefixes. Going back to our example of `prefix1_file1.csv` and `prefix2_file2.csv` having different schemas,
+lets suppose our storage container has a structure as follows:
+```
+directory_to_profile
+├── heterogeneous_subdirectory
+│   ├── prefix1_file1.csv
+│   ├── prefix1_file2.csv
+│   ├── prefix2_file1.csv
+│   └── prefix2_file2.csv
+└── homogeneous_subdirectory
+    ├── file1
+    ├── file1.csv
+    ├── file1.txt
+    ├── file2
+    ├── file2.csv
+    └── file2.txt
+```
+
```

Comment:
this is great!

Reply:
Thanks! It was easy to create using `mkdir`, `touch`, and `tree` - it does a fantastic job of showing directory structure.